### PR TITLE
Put orchestration specific feature in its own package

### PR DESCRIPTION
**Pull request by jemoreira**

### CODE REVIEW

File: host/frontend/operator/tags [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Mmain.go	main.go	15
```
Comment (ser-io) [inline host/frontend/operator/tags:1]: wondering what's this file for? is it vim tags?

File: host/frontend/liboperator/operator/operator.go [hunk 2]
```diff
@@ -28,11 +28,20 @@ import (
 	"github.com/gorilla/mux"
 )
 
+const (
+	DefaultSocketPath     = "/run/cuttlefish/operator"
```
Comment (ser-io) [inline host/frontend/liboperator/operator/operator.go:32]: I think is clearer if the binaries define their own default values. the problem I see here is that the library's users do not need to to use these defaults to use the library and the library does not use these values either so the library is exposing and being responsible of something that's not necessary needed.

File: host/frontend/operator/tags (hunk 1 above)
Comment (jemoreira) [reply host/frontend/operator/tags]: yes, removed it.

File: host/frontend/liboperator/operator/operator.go (hunk 2 above)
Comment (jemoreira) [reply host/frontend/liboperator/operator/operator.go]: done

**Review by Databean — APPROVED** [review]

**Pull request merged**